source/settings/core.rst

```diff
@@ -2295,11 +2295,16 @@ See :ref:`settings` for list of all setting groups.
    :default: 0
    :values: @uint
 
-   The maximum number of messages to keep open and prefetch to memory.
-
-   ``0`` indicates no limit should be applied.
-
-   Behavior is dependent on the operating system and mailbox format.
+   The number of messages to try to prefetch whenever possible. The exact
+   behavior depends on the mailbox format. This is mainly useful with the imapc
+   and the obox mailbox formats where multiple mails are read from a remote
+   storage into local disk without waiting for previous reads to finish.
+   Depending on the remote storage latency, this may significantly speed up
+   performance when reading many mails. The downside is that it may increase
+   memory and file descriptor usage. For imapc and obox formats a good value
+   is likely between 10..100.
```

Comment:
Explain further that it affects reads, writes, deletes, copies.. And link to obox-specific settings that override.

Reply:
I guess also mention posix_fadvise() as possibly making a difference with NFS and Maildir/sdbox..